scripts/git/update_to_llvm_syncpoint.py

```diff
@@ -61,20 +61,30 @@
 def parse_arguments():
```

Comment:
GitHub won't let me comment outside of changed lines... but the file comment should be updated:

```
In addition we currently copy LLVM Bazel BUILD files from TensorFlow.
...
  By default, this will update the TensorFlow submodule to the most recent
  commit with an LLVM version that matches IREE's and copy over the LLVM
  BUILD file changes as needed.
```

Reply:
Thanks for catching. Done.